Sources/App/configure.swift

```diff
@@ -4,24 +4,22 @@ import Stevenson
 /// Called before your application initializes.
 public func configure(_ config: inout Config, _ env: inout Environment, _ services: inout Services) throws {
     let slack = SlackService(
-        token: try attempt { Environment.get("SLACK_TOKEN") }
+        token: try attempt { Environment.get(.SLACK_TOKEN) }
     )
 
     let ci = CircleCIService(
-        project: try attempt { Environment.get("GITHUB_REPO_IOS") },
-        token: try attempt { Environment.get("CIRCLECI_TOKEN") },
-        defaultBranch: "develop"
+        token: try attempt { Environment.get(.CIRCLECI_TOKEN) }
     )
 
     let jira = JiraService(
-        host: try attempt { Environment.get("JIRA_HOST") },
-        username: try attempt { Environment.get("JIRA_USERNAME") },
-        password: try attempt { Environment.get("JIRA_TOKEN") }
+        baseURL: try attempt { Environment.get(.JIRA_HOST).flatMap(URL.init(string:)) },
```

Comment:
🤔 is a string containinig only a host name really a valid `URL`?

```swift
URL(string: "apple.com")!.host // nil
```

Note that the `JIRA_HOST` env var is supposed to contain just the host name, as the host name might be used either to create the REST API baseURL from it (fixing the scheme to `https://` and the port) or for other means like building the URL for the created ticket (not the same port as the REST API then)

Reply:
It should contain a proper string, so it's a programmer mistake if the string is invalid. The env variable should be updated with this of course, but I think the name is fine as it is (doc comment should be enough). Adding port to the base URL does not change anything when opening it in a browser.